### ISSUE TITLE
Should the failure to implement Sufficient Technique in the right way be treated as a WCAG failure

### ISSUE THREAD

**Issue by iamrafan** [w3c/wcag#318]
Ex:
[ARIA11: Using ARIA landmarks to identify regions of a page](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/ARIA11) is one of the sufficient techniques to meet the success criteria WCAG 1.3.1 and WCAG 2.4.1. 

Although it is not required to use this technique, if an author chooses to use this technique and fails to implement it in the right way, should we treat it as a WCAG failure?

**Comment by alastc** [comment]
It depends what you mean by 'right way'.

If they are used in a way that fulfils the criteria, then it passes. 

However, you could use them badly and still pass, e.g. represent the visual areas and provide a means of skipping repeated links. But you might miss certain areas of the page which don't have a visual meaning.

I can't think of a way off-hand that you could use them well and fail, but I might be lacking imaganation!

BTW, for a quicker response I'd recommend the [WAI interest group](https://www.w3.org/WAI/IG/) or [Webaim forum](https://webaim.org/discussion/). The issues here are for updating the documents, for general questions about how to apply, the email lists are better.

Kind regards,

-Alastair

**Comment by iamrafan** [comment]
@alastc thanks for the response. 

I meant, for example if an author uses landmarks(one of the sufficient techniques) to skip the repeated links but fails in implementing the landmarks according to the ARIA spec. 

Is it good enough to consider it as a WCAG failure as the author probably might not be using any other technique other than landmarks to skip links?

**Comment by alastc** [comment]
If a user cannot use the landmarks to skip blocks of links, and there is no other means of doing so, then it would not pass.

You could implement just enough landmarks to enable that, but not do it according to spec. If it works to skip links it passes that criteria (but might fail 1.3.1 if it doesn't convey the info correctly).

As this isn't an issue with the WCAG spec I'll close this, but please use the interest group or webaim links above for more.
